application/templates/static_site/static_pages/ethnic_groups.html

```diff
@@ -72,7 +72,7 @@ <h2 class="govuk-heading-s govuk-!-margin-bottom-1">Other ethnic group</h2>
       <p class="govuk-body">In Wales, the first option in the White broad category is changed so that Welsh appears first in the list, followed by English, Scottish, Northern Irish and British. </p>
 
 
-      <p class="govuk-body">See statistics on the <a class="govuk-link" href="/british-population/national-and-regional-populations/population-of-england-and-wales/latest">population of England and Wales</a> broken down by ethnicity.</p>
+      <p class="govuk-body">See statistics on the <a class="govuk-link" href="/british-population/national-and-regional-populations/population-of-england-and-wales/latest" data-on="click" data-event-category="External link clicked" data-event-action="Ethnic groups link" data-event-label="population of England and Wales">population of England and Wales</a> broken down by ethnicity.</p>
```

Comment:
This isn't an external link so we probably don't want to use `External link clicked` as the event category? Possibly we don't even need this event as navigation between internal pages is tracked by default (unless there are two links to the same page, in which case you can't tell which one was clicked).